stages/stage1.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# This stage builds packages from source repositories and
+# creates dnf repos of the results
+set -xeuo pipefail
+
+echo "# Installing required packages"
+dnf install -y git rsync openssh-clients dnf-plugins-core fedpkg dnf-utils awscli
+cp $WORKSPACE/RPM-GPG-* /etc/pki/rpm-gpg/
+dnf copr -y enable walters/buildtools-fedora
+dnf install -y rpmdistro-gitoverlay
```

Comment:
Feels like we should ultimately be modifying the `coreos-assembler` container to have these packages installed.

Reply:
@cgwalters WDYT? :arrow_up:

Reply:
https://github.com/cgwalters/coreos-assembler/pull/6